config/database.yml

```diff
@@ -40,8 +45,16 @@ development:
 # re-generated from your development database when you run "rake".
 # Do not set this db to the same as development or production.
 test:
-  <<: *default
-  database: ecom_site_test
+  primary:
+    <<: *default
+    url: <%= ENV["DATABASE_URL"] %>
```

Comment:
PAIN - Turns out that this is the default behaviour when there is 1 database. So have to manually specify to use `ENV["DATABASE_URL"]` for both